index.html

```diff
@@ -101,7 +96,252 @@ <h1>I'm Adeyemi Glad to see you!</h1>
                 </li>  
               </ul>            
             </div>
-            <div> 
+          </div>
+        </section>
+        <Section class="Works">
+          <div class="cardWork">
+            <img id="snap_portfolio">
+            <div class="leftBlock">
+              <div class="pryText_head">
+                <div class="text_head">
+                  Tonic
```

Comment:
Kindly put this in `h` tag. This is important in SEO search and also to draw in the reader and give a clear indication of the content on the page ☺